build/rpm/pmm2-client.spec

```diff
@@ -75,6 +75,7 @@ install -m 0755 bin/postgres_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exp
 install -m 0755 bin/mongodb_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
 install -m 0755 bin/proxysql_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
 install -m 0755 bin/rds_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
+install -m 0755 bin/mnogo_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
```

Comment:
@alexanderblinkov-percona  are we going to deliver both versions of mongodb_exporter?